src/libstd/Cargo.toml

```diff
@@ -47,6 +47,9 @@ backtrace = []
 panic-unwind = ["panic_unwind"]
 profiler = ["profiler_builtins"]
 
+# Make panics and failed asserts immediately abort without formatting any message
+panic_immediate_abort = []
```

Comment:
Should this enable the feature for libcore?

Reply:
I don't know, I added `core` line to `Xargo.toml` just in case. Does it need to be experimented?

Reply:
I think that if you enable it for libstd you almost for sure want to enable it for libcore, right? Worth testing both ways!

Reply:
Now I understand. I forgot for a moment that a feature can automatically turn on features in dependent crates.

`panic_immediate_abort = ["core/panic_immediate_abort"]`?